waterlog/src/Components/AlertBox.js

```diff
@@ -2,26 +2,26 @@ import React from 'react';
 
 function highStatus() {
 	return (
-		<svg width="15" height="15">
-			<rect x="0" y="0" width="15" height="15" style={{ fill: '#ff1744' }} />
+		<svg width="12" height="12">
+			<rect x="0" y="0" width="12" height="12" style={{ fill: '#ff1744' }} />
 			Sorry, your browser does not support inline SVG.
 		</svg>
 	);
 }
 
 function mediumStatus(props) {
 	return (
-		<svg width="15" height="15">
-			<rect x="0" y="0" width="15" height="15" style={{ fill: '#ffab00'}} />
+		<svg width="12" height="12">
+			<rect x="0" y="0" width="12" height="12" style={{ fill: '#ffab00'}} />
 			Sorry, your browser does not support inline SVG.
 		</svg>
 	);
 }
 
 function lowStatus(props) {
 	return (
-		<svg width="15" height="15">
-			<rect x="0" y="0" width="15" height="15" style={{ fill: '#ffea00' }} />
+		<svg width="12" height="12">
+			<rect x="0" y="0" width="12" height="12" style={{ fill: '#ffea00' }} />
```

Comment:
You could even keep the sizing constraints in a stylesheet of some sort and use that for your width and height of images etc. That way changing it in one place will change all that use the same value. If you feel like you're often re-using the same constraints

Reply:
Working on it

Reply:
Please review i am done some changes